lib/services/microsoft_teams.rb

```diff
@@ -205,8 +204,13 @@ def facts
 
         parsed.css('span.deleted').each { |node| node.name = 'strike' }
         parsed.css('span.inserted').each { |node| node.name = 'strong' }
-        obj["value"] = parsed.to_s
+
+        obj["value"] = html_to_markdown(parsed.to_s)
       end
     end
   end
+
+  def html?(string)
+    !!(string =~ /<[^>]*>/)
```

Comment:
Take it or leave it, but I think a guard clause is more readable than `!!`. Just makes me think to much.